src/Microsoft.ML.Data/Evaluators/Metrics/RankerMetrics.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Data
+{
+    public sealed class RankerMetrics
+    {
+        /// <summary>
+        /// Normalized Discounted Cumulative Gain
```

Comment:
>Normalized Discounted Cumulative Gain [](start = 12, length = 37)

Array of normalized discounted cumulative gains where i-th element represent NDCG@i (you can also phrase it as NDCG at i).